client/packages/electron/package.json

```diff
@@ -15,19 +15,20 @@
   "config": {
     "forge": {
       "packagerConfig": {
-        "icon": "/Users/andreievguenov/Downloads/check.icns"
+        "icon": "./src/public/oms"
```

Comment:
leaving the extension off allows auto detection of the required file ( according to the docs )